source/manual/get-emails-about-this-page.html.md

```diff
@@ -12,12 +12,10 @@ The [‘get emails about this page’ button][get-emails-about-this-page] was ro
 
 Before rolling out the feature, the GOV.UK account team contacted 14 departments who had content that offered users a work-around for subscribing to page-level updates.
 
-Departments asked for these 4 pages to be excluded from the roll-out:
```

Comment:
non-blocking nitpick: I wonder if we should say "Departments asked for 4 pages to be exluded, of which 2 now no longer exist. The remaining 2 are:" (which is more historically accurate).